care/facility/tasks/redis_index.py

```diff
@@ -15,6 +16,13 @@
 
 @shared_task
 def load_redis_index():
+    try:
+        deleted_count = BaseCommand.delete_pattern("care_static_data*")
```

Comment:
```suggestion
        deleted_count = cache.delete_pattern("care_static_data*")
```

Reply:
Again, thank you for the feedback. I've changed it to...
```deleted_count = cache.delete_pattern("care_static_data*", itersize=25_000)```